dub.json

```diff
@@ -11,5 +11,12 @@
 	"sourcePaths": ["source"],
 	"importPaths": [
 		"source"
-	]
+	],
+	"buildTypes": {
+		"unittest-dip1000": {
```

Comment:
You should add this to `configurations` instead. This permits other users to use the preview in their code with `subConfigurations`.

Reply:
I don't understand.